### PR TITLE
#60 show/hide stereo flags base on option value + refactor to ts

### PR DESCRIPTION
**Pull request by ensemenova**

### CODE REVIEW

File: packages/ketcher-react/src/script/render/restruct/ReEnhancedFlag.ts [hunk 1]
```diff
@@ -14,34 +14,41 @@
...
+  constructor(flag: StereoFlag | null, pos: Vec2 | null) {
     super('enhancedFlag')
     this.flag = flag
     this.pp = pos
   }
   static isSelectable() {
     return true
   }
-  highlightPath(render) {
+  highlightPath(render: any): any {
```
Comment (TomasLight) [inline packages/ketcher-react/src/script/render/restruct/ReEnhancedFlag.ts:33]: please, use `Render` class for type `render` argument

File: packages/ketcher-core/src/chemistry/entities/Fragment.ts [hunk 2]
```diff
@@ -13,30 +13,30 @@
...
-function calcStereoFlag(struct: any, stereoAids: Array<number>) {
+export enum StereoFlag {
+  Mixed = 'Mixed',
+  abs = 'ABS (Chiral)',
+  and = 'AND Enantiomer',
+  or = 'OR Enantiomer'
+  // todo: custom in the future
+}
+
+function calcStereoFlag(struct: any, stereoAids: Array<number>): string | null {
```
Comment (TomasLight) [inline packages/ketcher-core/src/chemistry/entities/Fragment.ts:24]: and `Struct` (from 'ketcher-core') for `struct` argument

**Review by TomasLight — CHANGES_REQUESTED** [review]

File: packages/ketcher-core/src/chemistry/entities/Fragment.ts (hunk 2 above)
Comment (ensemenova) [reply packages/ketcher-core/src/chemistry/entities/Fragment.ts]: fixed

File: packages/ketcher-react/src/script/render/restruct/ReEnhancedFlag.ts (hunk 1 above)
Comment (ensemenova) [reply packages/ketcher-react/src/script/render/restruct/ReEnhancedFlag.ts]: fixed

File: packages/ketcher-core/src/chemistry/entities/Fragment.ts [hunk 3]
```diff
@@ -13,30 +13,35 @@
...
-    and: 'AND Enantiomer',
-    or: 'OR Enantiomer',
-    null: null
-    // todo: custom in the future
-  }
 
+export class Fragment {
   stereoAtoms: Array<number>
-  enhancedStereoFlag?: any
+  enhancedStereoFlag?: boolean | string | undefined
```
Comment (AndreiMazol) [inline packages/ketcher-core/src/chemistry/entities/Fragment.ts:42]: enhancedStereoFlag is already marked as undefined ('?' mark) so no needs to use union with 'undefined' explicitly

File: packages/ketcher-react/src/script/render/restruct/ReObject.ts [hunk 4]
```diff
@@ -28,15 +30,15 @@ class ReObject {
     this.visel = new Visel(viselType)
   }
 
-  getVBoxObj(render: any): any {
+  getVBoxObj(render: Render): any {
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/render/restruct/ReObject.ts:33]: You can return 'Box2Abs' instead of 'any'

**Review by AndreiMazol — APPROVED** [review]

**Review by TomasLight — APPROVED** [review]

**Pull request merged**